INSTALL.md

```diff
@@ -334,6 +334,13 @@ sudo apt-get install postgresql-contrib
 # or: sudo apt-get install postgresql-contrib-9.3 depending on your OS and apt repository versions
 ```
 
+### Apple Silicon Mac (M1/M2)
+
+#### `no such file or directory` or `fatal error: file not found`
+
+M1 and M2 macs have a new feature for homebrew where homebrew is not installed to the path `usr/local/bin` like Intel Macs are, but to `opt/homebrew/bin`. Since some old code likely hasn't been updated yet, this error could possibly be from the code looking into the old Intel Mac path. Make that homebrew libraries are being searched for in the `opt/homebrew/lib` path.
```

Comment:
Be assertive. "Since some old code likely hasn't been updated yet, this error could possibly be from the code looking into the old Intel Mac path." is so cautiously worded that it doesn't inspire faith in the installation document. 

It's good to attribute sources to buttress your assertions, but in this case I think it would be better if you paraphrased the relevant information from the URL and placed it here as well as attributing the source.

In part this is because this URL is to a third-party source that could easily disappear and important information would then be lost. Secondly, we should provide enough relevant information to the user so that traversing the link and processing an entire additional document is optional, rather than a requirement.